Update ocr.rst (#1653)
odoo/documentation: content/applications/finance/accounting/payables/supplier_bills/ocr.rst

==============================================================

Encoding bills manually can be a time-consuming task. Having a solution that allows you to digitize
them and automatically import the data into your database reduces errors and saves you time.

.. important::
   - Try it out! Upload one of your Bills or try one of our samples.
   - Click `here <https://www.odoo.com/app/invoice-automation>`_ to evaluate it now!
   
Set up the feature
==================

Go to :menuselection:`Accounting --> Settings --> Bill Digitalization`, and choose whether the bills

> should be processed automatically (use the OCR service every time) or manually (on demand).

Reviewer: ```suggestion
should be processed automatically ( with the OCR ) or manually.
```
